toolsrc/src/tls12-download.c

```diff
@@ -0,0 +1,263 @@
+#include <Windows.h>
+#include <process.h>
+#include <winhttp.h>
+/*
+ * This program must be as small as possible, because it is committed in binary form to the
+ * vcpkg github repo to enable downloading the main vcpkg program on Windows 7, where TLS 1.2 is
+ * unavailable to PowerShell.
+ * To that end it avoids using C runtime functions (beyond the vcruntime ones the compiler
+ * injects itself).
+ * (In testing as of 2021-01-07, this version that doesn't link with the CRT is ~8kb, whereas a
+ * hello world program that does link with the CRT is ~300kb)
+ */

```

Comment:
```suggestion
    This program must be as small as possible, because it is committed in binary form to the
    vcpkg github repo to enable downloading the main vcpkg program on Windows 7, where TLS 1.2 is
    unavailable to PowerShell.
    To that end it avoids using C runtime functions (beyond the vcruntime ones the compiler
     injects itself).
     (In testing as of 2021-01-07, this version that doesn't link with the CRT is ~8kb, whereas a
    hello world program that does link with the CRT is ~300kb)
*/
```

this is the standard way we do block comments (or at least it's the one I use)

Reply:
We use *-per-line comments in the code, such as https://github.com/microsoft/vcpkg/blob/9c63f97fa1d4915a7dfc5f32d5cdb6ee8fe9bf8b/toolsrc/src/vcpkg/vcpkgpaths.cpp#L708-L715

I prefer the code as written.

Reply:
hmm, alright, I misremembered then.